### PR TITLE
Ui improvements

### PR DESCRIPTION
**Pull request by wbelhomsi**
Added variables of css
Completed Circle

https://github.com/wbelhomsi/emergent-lebanon/issues/2

### CODE REVIEW

File: frontend/src/components/explore-page/explore-page.component.js [hunk 1]
```diff
@@ -7,8 +7,35 @@ import Loader from '../loader/loader.component';
...
+        selected={selectedFilter === circleComp.id}
+        onClick={() => setSelectedFilter(circleComp.id)}
+      />
+    )
+  })
+}
+
 const ExplorePage = () => {
   const dispatch = useDispatch();
+  const [selectedFilter, setSelectedFilter] = useState(-1);
```
Comment (mallocassi) [inline frontend/src/components/explore-page/explore-page.component.js:25]: Won't this filter all Circles by default?

**Review by mallocassi — APPROVED** [review]
I'm not entirely sure what the desired filtering capabilities are, but for now it looks good to me!

File: frontend/src/components/explore-page/explore-page.component.js (hunk 1 above)
Comment (wbelhomsi) [reply frontend/src/components/explore-page/explore-page.component.js:25]: This is for the when clicking a circle, by default i want no circle to be selected, so the value of selectedFilter is `-1`, when clicking a circle, we trigger setSelectedFilter which in tern turns selectedFilter to the id and highlights it

**Comment by wbelhomsi** [comment]
@mallocassi those filters will change the pins on the maps, so depending on the selected filter, pins will be shown or areas will be highlighted

**Pull request merged**